lib/renderer.coffee

```diff
@@ -14,17 +14,25 @@ highlighter = null
 packagePath = path.dirname(__dirname)
 
 exports.toHtml = (text='', filePath) ->
+  render text, filePath
+    .then (html) ->
+      sanitize html
+    .then (html) ->
+      resolveImagePaths html, filePath
+    .then (html) ->
+      tokenizeCodeBlocks html
```

Comment:
Not here because this line is use by the preview. The 'save as' use `toRawHtml` method.